### PR TITLE
docs: add note about bubblejail and chromium

### PR DESCRIPTION
**Pull request by RKNF404**

### CODE REVIEW

**Review by TheHooly — DISMISSED** [review]

**Review by RoyalOughtness — DISMISSED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -134,7 +134,9 @@ mkdir -p ~/.config/environment.d && echo "GSK_RENDERER=gl" >> ~/.config/environm
 This should no longer be required as of F41: https://discussion.fedoraproject.org/t/gdk-message-error-71-protocol-error-dispatching-to-wayland-display/127927/42
 
 #### Why won't `hardened-chromium` start?
-Try starting `hardened-chromium` from the commandline by running `chromium-browser`. If you get an error about the current profile already running on another device, this is an issue with upstream chromium which can happen when you `rpm-ostree update` or `rpm-ostree rebase`. To fix this, simply run `rm ~/.config/chromium/Singleton*`
+Try starting `hardened-chromium` from the commandline by running `chromium-browser`. If you get an error about the current profile already running on another device, this is an issue with upstream chromium which can happen when you `rpm-ostree update` or `rpm-ostree rebase`. To fix this, simply run `rm ~/.config/chromium/Singleton*`.
```
Comment (mintpilo) [inline docs/FAQ.md]: Often in my experience you don't need to remove all Singleton* files, just SingletonLock.

Comment (RoyalOughtness) [reply docs/FAQ.md]: good point

Comment (RKNF404) [reply docs/FAQ.md]: Are there other Singleton files?

Comment (mintpilo) [reply docs/FAQ.md]: SingletonCookie, SingletonSocket, etc...

Comment (RKNF404) [reply docs/FAQ.md]: Hmm

**Review by mintpilo — APPROVED** [review]

**Review by RoyalOughtness — APPROVED** [review]

**Pull request merged**